meta-xt-prod-cockpit-rcar-control/recipes-guests/doma/doma.bbappend

```diff
@@ -31,3 +31,10 @@ do_install_append() {
         install -m 0644 ${WORKDIR}/doma-pvcamera.conf ${D}${sysconfdir}/systemd/system/doma.service.d
     fi
 }
+
+do_install_append_scmi() {
```

Comment:
Hi @iusyk, Please move this part to the do_install_append section above so that we do not rely on the same OVERRIDE we currently do not want to use. Let's add it later when we create a working yaml ENABLE_SCMI yaml parameter.

Reply:
Done.